src/patches/vs1053b-patches-flac-latm.plg

```diff
@@ -0,0 +1,1142 @@
+/* User application code loading tables for VS10xx */
+#if 0
+void LoadUserCode(void) {
```

Comment:
Do we really need to include `LoadUserCode(void)` everywhere which we will not use. This has a pseudo-code like `WriteVS10xxRegister` which doesn't work for sure, and we have our own `VS1053::loadUserCode(const unsigned short* plugin)`.